uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request.controller.js

```diff
@@ -491,9 +491,15 @@ define([
               $rootScope.$emit('LeaveRequestPopup::absencePeriodBalancesUpdated', vm.absenceTypes);
             });
         }),
-        $rootScope.$on('LeaveRequestPopup::handleError', function (__, errors) { handleError(errors); }),
-        $rootScope.$on('LeaveRequestPopup::childComponent::register', function () { childComponentsCount++; })
+        $rootScope.$on('LeaveRequestPopup::handleError', function (__, errors) {
```

Comment:
I think it was me, who wrote the function in one line after @igorpavlov 's comment. Should we consider not writing in a single line as a style guide item?

Reply:
yes, i will add it